CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ Changelog
 
 Old Changes:  https://sbprojects.statsbiblioteket.dk/display/INFRA/Yousee+Channel+Archiving+Requester+Releases
 
+Version 1.1.12
+=============
+Bugfix update
+
+ 1. Fix parsing of dates when no time information is present (i.e. allow to changes dates for requests)
```

Comment:
Something off about this sentence: "(i.e. allow to changes dates for requests)", perhaps "allow to change dates for requests" or "allow changing dates for requests"?